project_task_wbs/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Project Task WBS
+================
+
+This module extends the functionality of Project and allow you to add a WBS column to project and task.
```

Comment:
Check the OCA if there is another similar module.

This module extends the functionality of Project and adds a Work Break Structure (WBS) field to tasks.